spec/lib/papertrail_spec.rb

```diff
@@ -65,18 +65,18 @@ def type_abbreviation
       end
     end
 
-    it 'does not restore the gov_uk_dates' do
+    it 'restores the gov_uk_dates' do
```

Comment:
Great!  I think we need to update the readme too on this - I think it specifically says that gov_uk dates don't work and gives a work-around.